webdriver/client.py

```diff
@@ -317,21 +317,20 @@ def start(self):
         #body["capabilities"] = caps
         body = caps
 
-        response = self.send_raw_command("POST", "session", body=body)
+        value = self.send_command("POST", "session", body=body, requires_session=False)
 
-        self.session_id = response.body["value"]["sessionId"]
+        self.session_id = value["sessionId"]
 
         if self.extension_cls:
             self.extension = self.extension_cls(self)
 
-        return response.body["value"]
+        return value
```

Comment:
Although this matches the phrasing of the original implementation, it might be more useful to return `value["sessionId"]` here instead.